### PR TITLE
fix: error for multi-replica volume from snapshot

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
FIX: The error variant for multi replica snapshot clone/restore creation was the same as multi replica snapshot operations. Add a specific variant for the same.

### CODE REVIEW

File: control-plane/agents/src/common/errors.rs [hunk 1]
```diff
@@ -298,6 +298,8 @@ pub enum SvcError {
     },
     #[snafu(display("Snapshots for multi-replica volumes are not allowed"))]
     NReplSnapshotNotAllowed {},
+    #[snafu(display("Multi-replica volume is not allowed from snapshot"))]
```
Comment (avishnu) [inline control-plane/agents/src/common/errors.rs]: Cannot create a multi-replica volume from a snapshot of a single-replica volume

Comment (Abhinandan-Purkait) [reply control-plane/agents/src/common/errors.rs]: Are you suggesting the belongs to be added at the end?

Comment (avishnu) [reply control-plane/agents/src/common/errors.rs]: > Are you suggesting the belongs to be added at the end?

Just suggesting a modified text.

Comment (Abhinandan-Purkait) [reply control-plane/agents/src/common/errors.rs]: Sounds good. Done. Thanks

Comment (avishnu) [reply control-plane/agents/src/common/errors.rs]: Just wondering if the message should be generic, something like "Not enough sources to create the volume replicas".
We can probably take this up later.

**Review by avishnu — APPROVED** [review]
Lgtm

File: control-plane/agents/src/common/errors.rs (hunk 1 above)
Comment (Abhinandan-Purkait) [reply control-plane/agents/src/common/errors.rs]: Sure.

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**